### ISSUE TITLE
Override values with resolvers (like type-graphql behavior)

### ISSUE THREAD

**Issue by smmoosavi** [nestjs/graphql#219]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

Resolver does not work as expected like type-graphql

resolver:
```ts
@Resolver(of => NodeType)
export class NodeResolver {
  @ResolveProperty('id', returns => ID, { name: 'id' })
  async globalId(@Root() node: NodeType, @Info() info: GraphQLResolveInfo) {
    return toGlobalId(node.id, info.parentType);
  }
}
```
other classes:
```ts
export const toGlobalId = (id: string | number, type: NamedType) => {
  return Base64.encodeURI(type.name + ':' + id);
};

@ObjectType()
@InterfaceType('Node')
export class NodeType {
  @Field(type => ID)
  readonly id: string | number;
}

function Node() {
  return ObjectType({ implements: NodeType });
}

@Node()
export class ViewerNode extends NodeType {}

/* and many other classes extend NodeType */

@Resolver()
export class ViewerResolver {
  @Query(returns => ViewerNode)
  async viewer() {
    return ofType(ViewerNode)({ id: '0' })
  }
}

// https://github.com/19majkel94/type-graphql/issues/73#issuecomment-386833394
export function ofType<T>(Type: new (...args: any[]) => T) {
  return (object: T) => {
    return Object.assign(new Type(), object);
  };
}

```
query: 
```graphql
query{
  viewer{
    id
  }
}
```
response:
```json
{
  "data": {
    "viewer": {
      "id": "0"
    }
  }
}
```
## Expected behavior

the resolver applied to all classes extends NodeType. and have output like this:

```json
{
  "data": {
    "viewer": {
      "id": "Vmlld2VyTm9kZTow"
    }
  }
}
```
type-graphql behavior had this behavior

## Minimal reproduction of the problem with instructions

 see code :arrow_up: 
## What is the motivation / use case for changing the behavior?

type-graphql behavior had this behavior

## Environment

<pre><code>
"@nestjs/common": "^6.1.1",
"@nestjs/core": "^6.1.1",
"@nestjs/graphql": "^6.0.5",

 
For Tooling issues:
- Node version: v10.4.1
- Platform:  Linux

Others:

"type-graphql": "^0.17.1",

</code></pre>

**Comment by kamilmysliwiec** [comment]
It should be fixed in the @next version (`$ npm i @nestjs/graphql@next`). You will also have to update `@nestjs/core` to 6.2.0 (`$ npm i @nestjs/core@6.2.0`)

**Comment by kamilmysliwiec** [comment]
Published as `6.2.0` (`$ npm i @nestjs/graphql@latest`). You will also have to update `@nestjs/core` to 6.2.0 (`$ npm i @nestjs/core@latest`)
